tests/blas1_interface_test.cpp

```diff
@@ -63,47 +92,39 @@ int main(int argc, char *argv[]) {
     double nrmX = 0.0, nrmY = 0.0, max = 0.0, min = 1e9;
     double diff = 0.0;
     double _cos, _sin, giv = 0.0;
-    std::for_each(std::begin(vZ), std::end(vZ), [&](double &elem) {
-      elem = vY[i] + alpha * vX[i];
-      sum += std::abs(elem);
-      dot += (elem * vX[i]);
-      nrmX += vX[i] * vX[i];
-      nrmY += elem * elem;
-      if (std::abs(elem) > std::abs(max)) {
-        max = elem, indMax = i;
-      }
-      if (std::abs(elem) < std::abs(min)) {
-        min = elem, indMin = i;
-      }
-      if (i == 0) {
-        diff = elem - vX[i];
-        double num1 = vX[0], num2 = elem;
-        blas::_rotg(num1, num2, _cos, _sin);
-      }
-      giv += ((vX[i] * _cos + elem * _sin) * (elem * _cos - vX[i] * _sin));
-      if (i == 0) {
-        diff = (elem * _cos - vX[i] * _sin) - ((vX[i] * _cos + elem * _sin));
-      } else if ((i + 1) == sizeV) {
-        diff += (elem * _cos - vX[i] * _sin) - ((vX[i] * _cos + elem * _sin));
-      }
-      i++;
-    });
-    nrmX = std::sqrt(nrmX);
-    nrmY = std::sqrt(nrmY);
+    /* std::for_each(std::begin(vZ), std::end(vZ), [&](double &elem) { */
```

Comment:
These computation are useful to verify the correct execution of the SYCLBLAS interface